README.md

```diff
@@ -130,6 +130,9 @@ pnpm run dev
 | [<img height="50" src="https://raw.githubusercontent.com/ckb-ecofund/ccc/master/assets/projects/nervape.svg" />](https://www.nervape.com/) | [<img height="50" src="https://raw.githubusercontent.com/ckb-ecofund/ccc/master/assets/projects/utxoglobal.svg" />](https://utxo.global/) | [<img height="50" src="https://raw.githubusercontent.com/ckb-ecofund/ccc/master/assets/projects/boolnetwork.svg" />](https://bool.network/) | [<img height="50" src="https://raw.githubusercontent.com/ckb-ecofund/ccc/master/assets/projects/joydao.svg" />](https://joydao.cc/) | [<img height="50" src="https://raw.githubusercontent.com/ckb-ecofund/ccc/master/assets/projects/world3.svg" />](https://world3.ai/) |
 | ------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
 
+| [<img height="50" src="https://imagedelivery.net/zmmqprAdW82mq5ue46Pk1Q/a41719ca-1d43-4531-bdff-f602c28fd500/public" />](https://rgbcat.io/) |
```

Comment:
The rgbcat website doesn't have things related to CCC. Please update the correct link here or add the link to your rgbcat website.

Also, like other icons, please include the icon image file in the repo.